redcap_bridge/test_redcap/test_cli.py

```diff
@@ -30,23 +30,23 @@ def test_download(initialize_test_dir):
     output_file = test_directory / 'cli_download_test.csv'
 
     # download with default arguments
-    result = subprocess.run(['RedCapBridge', 'download', output_file, SERVER_CONFIG_YAML],
+    result = subprocess.run(['RedCapBridge', 'download', output_file, SERVER_CONFIG_YAML, '--server', 'redcap'],
```

Comment:
You can't put optional argument after mandatory ones. I think in this case they will just be ignored.
```suggestion
    result = subprocess.run(['RedCapBridge', 'download', output_file, SERVER_CONFIG_YAML, '--server', 'redcap'],
```